.github/workflows/code_checks.yaml

```diff
@@ -0,0 +1,39 @@
+# .github/workflows/code_checks.yaml
+name: Code_Checks
+
+on: ["push", "pull_request"]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.2', '7.3', '7.4', '8.0', '8.1']
+        stability: [ prefer-lowest, prefer-stable ]
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }} tests
+    steps:
+      # basically git clone
+      - uses: actions/checkout@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache/files
+          key: dependencies-php-${{ matrix.php }}-composer-${{ hashFiles('composer.json') }}
+
+      # use PHP of specific version
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: pcov, dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite
+          coverage: pcov # none, disable xdebug, pcov
+
+      - name: Install dependencies
+        #run: composer update --${{ matrix.stability }} --prefer-dist --no-interaction
+        # To install laminas/code Version 4.5.x with php 8.1 support
+        run: composer update --${{ matrix.stability }} --prefer-dist --no-interaction --with-all-dependencies
+
+      - name: Execute tests
+        run: vendor/bin/phpunit --verbose
```

Comment:
I'd love to see the other tests we run on Travis also being run on GH Actions. We utilize Ant for running tests, see the build.xml. I do not propose using Ant but maybe we can put some energy into moving the test set to the Composer scripts section?

Reply:
I have never used Ant or Travis. PHP-CS-Fixer from EasyAdminBundle was working fine.
I searched for your github actions. Found this one https://github.com/OpenConext/OpenConext-engineblock/blob/master/.github/workflows/test-integration.yml
Will try to integrate more quality checks

Reply:
I moved the 5 scripts from the build file to composer-script section. But i need to test if i did this correct.
Not that this is run after composer upgrade at the end-user.

Reply:
Look like it works fine. 
a ) Checks are performed. [See Testrun](https://github.com/Chris53897/Monitor-bundle/runs/3780911967?check_suite_focus=true) under "Install dependencies"
b) Checks do not run if bundle is required in a project. See below.

But of course this can be changed to somethin else. Please let me know.

`statusboard composer update
Loading composer repositories with package information                                                                                                  Updating dependencies                                 
Lock file operations: 0 installs, 1 update, 0 removals
  - Upgrading openconext/monitor-bundle (dev-feature/symfony5 51896e2 => dev-feature/symfony5 417fb28)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Downloading openconext/monitor-bundle (dev-feature/symfony5 417fb28)
  - Upgrading openconext/monitor-bundle (dev-feature/symfony5 51896e2 => dev-feature/symfony5 417fb28): Extracting archive
Generating autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
94 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

What about running composer global require symfony/thanks && composer thanks now?
This will spread some 💖  by sending a ★  to the GitHub repositories of your fellow package maintainers.

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [OK]
Executing script assets:install --symlink --relative public [OK]
`